baseplate/frameworks/pyramid/csrf.py

```diff
@@ -108,26 +116,30 @@ def get_csrf_token(self, request):
         """
         return request.params.get(self._param)
 
-    def check_csrf_token(self, request, supplied_token):
+    def check_csrf_token(self, request: Any, supplied_token: str) -> bool:
         """Return True if the supplied_token is valid.
 
         This is called automatically by Pyramid if you have configured it
         to require CSRF.
         """
         try:
-            token_version, sep, signature = supplied_token.partition(".")
+            version_str, sep, signature = supplied_token.partition(".")
+            token_version = int(version_str)
```

Comment:
explicitly int-ified first so the type signature of `_make_csrf_token_payload` could always take an int